Meta/Lagom/Tools/CodeGenerators/LibWeb/GenerateCSSPropertyID.cpp

```diff
@@ -595,7 +595,7 @@ bool is_animatable_property(PropertyID property_id)
 
 bool is_inherited_property(PropertyID property_id)
 {
-    if (property_id >= first_inherited_shorthand_property_id && property_id <= last_inherited_longhand_property_id)
+    if (property_id >= first_inherited_shorthand_property_id && property_id <= last_inherited_shorthand_property_id)
```

Comment:
Could this error have been exposed with a test? I.e. testing with a property between last_inherited_shorthand_property_id and `first_inherited_longhand_proeprt_id`?

Reply:
What's the issue?

Reply:
I cannot figure out how to properly expose/check is_inherited_property from CSS/HTML/JS

Reply:
That sounds like the best approach so far. Done